examples/ML/pandas_ddp_example.py

```diff
@@ -145,12 +82,18 @@ def validation(model, val_loader, loss_fn):
         )
     with torch.no_grad():
         for input_id, mask, val_label in val_loader:
-            output = model(input_id, mask)
+            input_id = input_id.to(device, non_blocking=True)
+            mask = mask.to(device, non_blocking=True)
+            val_label = val_label.to(device, non_blocking=True)
+
+            output = model(input_id, token_type_ids=None,
+                                    attention_mask=mask,
+                                    labels=val_label)
 
-            batch_loss = loss_fn(output, val_label.long())
+            batch_loss = output.loss
             
             # Track local loss and accuracy stats
-            acc = (output.argmax(dim=1) == val_label).sum().item()
+            acc = (output.logits.argmax(dim=1) == val_label).sum().item()
             total_ddp_acc_loss_train[0] += acc
             total_ddp_acc_loss_train[1] += batch_loss.item()
```

Comment:
I think this is still the average loss (similar change needed in train loop):
```suggestion
            total_ddp_acc_loss_train[1] += batch_loss.item() * BATCH_SIZE
```